src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -1,75 +1,41 @@
-import React from 'react';
-import { NavLink, useLocation } from 'react-router-dom';
+import React, {useState, useCallback} from 'react';
 
+import { Logo } from 'ui';
 import { Button } from '@material-ui/core';
```

Comment:
Enter - check readme file and section import order - will be much easier

Reply:
Formatting has been improved